CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Improvements:
 
 Bug Fixes:
 - Check if "CMakeLists.txt" exists after renaming. [#2986](https://github.com/microsoft/vscode-cmake-tools/issues/2986)
+- CMake kits chokes when parsing exported functions after running environmentSetupScript. [#2676](https://github.com/microsoft/vscode-cmake-tools/issues/2686)
```

Comment:
I know the customer used the word "chokes", but for the changelog we should change it to "fails"